.github/workflows/build.yml

```diff
@@ -0,0 +1,91 @@
+name: IntuneAppBuilder Deployment
```

Comment:
last nitpick - would just remove the name - it's self evident from the file name

Reply:
I think this is required for GitHub Actions to properly render the name to the UI, no?

Reply:
I figured it would just default back to "build" - if you want to make the name "Build" that's fine too